### DISCUSSION TITLE
CBCL Scoring Instructions

### DISCUSSION THREAD

**Discussion by cavanvauxbonner** [now-i-know-my-abcd/docs#76] [Where in the World is…? (Help)] [answered]
Hello! I am working on a project where we are using the T-Scores from the Child Behavior Checklist (CBCL) in the ABCD to derive clinical and borderline cutoffs. Unfortunately to scoring for the CBCL is proprietary. Is there any way for researchers who already have access to the ABCD data to access the scoring instructions? We have been able to glean what the scoring guidelines likely are, based on published papers, but it would be nice to base our analyses on the official guidelines just to cover all of our bases.

**Comment by abaskinsommers** [comment, marked as answer]
We are unable to release that information because it is proprietary. However, you can go to the ASEBA website and download a sample report which might give you that information (e.g., https://aseba.org/wp-content/uploads/cbclprofile.pdf).

**Comment by angielaird** [comment]
Awesome, thank you thank you! 😁On Jun 6, 2024, at 5:05 PM, abaskinsommers ***@***.***> wrote:﻿
We are unable to release that information because it is proprietary. However, you can go to the ASEBA website and download a sample report which might give you that information (e.g., https://aseba.org/wp-content/uploads/cbclprofile.pdf).
